### ISSUE TITLE
Users can view a website for this tool.

### ISSUE THREAD

**Issue by ChinmayPatel** [apigarage/atom-pomodoro-app#5]
Demonstrate what an app does (multiple GIFs)
Showcase contributors
Attribution to API Garage for gathering the resources

**Comment by MajinBui** [comment]
Other than the gifs(which I have a decent amount created already), does this require anything else?

**Comment by apigarage** [comment]
Yes, one major piece is needed -- your picture (or anything you wanted to be
identified with).
Remember, our goal is to put people over product.

**Comment by MajinBui** [comment]
Awesome, any specifics for the dimensions of the gifs?
